style: adjust template of new PR (#5193)
dashpay/dash: .github/PULL_REQUEST_TEMPLATE.md

> ***Please remove the following help written in cursive before merging***

Reviewer: ```suggestion
***Please remove the italicized help prompts before submitting or merging***
```
